privaterelay/locales/en-us/privaterelay/app.ftl

```diff
@@ -0,0 +1,226 @@
+
+## Profile Page (Logged In) - https://dev.fxprivaterelay.nonprod.cloudops.mozgcp.net/accounts/profile/
+
+# Variables:
+#   $email (string) - User email address
+profile-label-welcome = Welcome, { $email }!
+profile-headline-manage-relay = Manage your { -brand-name-relay } aliases
+profile-headline-manage-domain = Manage your Domain aliases
+profile-supports-email-forwarding = { -brand-name-firefox-relay } supports email forwarding (including attachments) of email up to 150KB in size
+profile-promo-premium-headline = Go premium for more features.
+profile-promo-premium-copy = With { -brand-name-relay } Premium, you get your own email domain.
+profile-promo-premium-cta = Buy Premium
+profile-label-saved = Label saved!
+profile-label-generate-new-alias = Generate New Alias
+profile-label-delete = Delete
+
+# This string is followed by an email address
+profile-label-forward-emails = Forward emails to:
+
+# This string is followed by date
+profile-label-first-emailed = First emailed:
+
+# This string is followed by date:
+profile-label-created = Created:
+profile-label-details-show = Show Details
+profile-label-details-hide = Hide Details
+profile-label-forwarding = forwarding
+profile-label-blocking = blocking
+profile-label-copied = Copied!
+profile-label-blocked = Blocked
+profile-label-forwarded = Forwarded
+profile-label-cancel = Cancel
+profile-blocked-copy = { -brand-name-firefox-relay } will delete messages before they can reach your inbox when you select blocking for this alias.
+profile-forwarded-copy-html = { -brand-name-firefox-relay } will send messages to your inbox when you select forwarding for this alias. <br />
+    <span class="fwd-note"><span class="bold ff-Met">Note:</span> Email (including attachments) larger than 150KB are not currently supported and will not be forwarded.
+
+## Banner Messages (displayed on the profile page) - https://dev.fxprivaterelay.nonprod.cloudops.mozgcp.net/accounts/profile/
+
+banner-bounced-headline = { -brand-name-relay } couldn’t deliver your email.
+
+# Variables:
+#   $username (string) - Username
+#   $bounce_type (string) - Type of bounced email
+#   $date (string) - A date for the next time the services tries to resend the email
+banner-bounced-copy = We are currently unable to send email to { $username }. 
+    We received a <em>{ $bounce_type }</em> “bounce” from your email provider when trying to forward emails to you. 
+    This can happen if { -brand-name-relay } couldn’t connect to your email provider, or if your mailbox was full. We will try again on { $date }.
+banner-download-firefox-headline = { -brand-name-relay } is even better in { -brand-name-firefox }
+banner-download-firefox-copy = The { -brand-name-relay } extension for { -brand-name-firefox-browser } makes creating aliases even easier.
+banner-download-firefox-cta = Get { -brand-name-firefox }
+banner-download-install-add-on-headline = Looks like you haven’t installed the add-on…yet!
```

Comment:
Q: Should there be a space between the "&hellip;" and "yet!"?

Reply:
Context for the string. I think we should remove it too. 

![image](https://user-images.githubusercontent.com/2692333/118539665-500a9880-b715-11eb-8b06-0a0afa9d89de.png)

Reply:
Revised: 
![image](https://user-images.githubusercontent.com/2692333/118539793-78929280-b715-11eb-836e-d9aa14ef3a75.png)